graylog2-server/src/main/java/org/graylog2/rest/resources/users/UsersResource.java

```diff
@@ -609,13 +611,25 @@ private UserSummary toUserResponse(User user,
             lastActivity = session.getLastAccessTime();
             clientAddress = session.getHost();
         }
+        List<WildcardPermission> wildcardPermissions;
+        List<GRNPermission> grnPermissions;
+        if (includePermissions) {
+            wildcardPermissions = userService.getPermissionsForUser(user).stream()
```

Comment:
How about creating `UserService#getWildcardPermissionsForUser` and `UserService#getGRNPermissionsForUser` to avoid duplication of this code block in `RolesResource`?

Reply:
good point. will do :+1: